AENS.md

```diff
@@ -125,7 +125,19 @@ This can have multiple entries, e.g. a payment address associated
 with the name and an oracle address associated with the name.
 
 
-### Name
+Entry for **subname** (or, subdomain) uses all the fields from
```

Comment:
What is a "subname"?

Is a subname an entity already included - via the other transactions - in the Fortuna release of AENS? (Clarifying this may require you to open a separate preparatory PR against master in which you detail Fortuna AENS.)

Reply:
Clarified in latest commit.